### DISCUSSION TITLE
Put Back "Open" or "Closed" Word in the Bullet List in the Meeting Detail

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#442] [General]
It's great that in v3.11.2 the text for open and closed meetings has been expanded (see #330), but am wondering whether you can put back the word open or closed in the bullet list displayed in the meeting detail page. 

Thanks

**Comment by tech2serve** [comment]
They show up on my site:

<img width="320" alt="image" src="https://user-images.githubusercontent.com/31904316/125335077-fe3f5280-e300-11eb-8776-c22e3f1c495c.png">

<img width="308" alt="image" src="https://user-images.githubusercontent.com/31904316/125335209-24fd8900-e301-11eb-9d52-7975f056a5d2.png">

**Comment by Grateful-Viv** [comment]
I was mistaken. Can you close this out. Thanks.
